page-components/half-width-link-blob.md

```diff
@@ -28,12 +22,13 @@ A pair of half-width link blobs, as seen on a landing page template:
 
 ## Use case
 <div class="content-67 content-first">
-##### When to use 
-- Default option for providing succinct descriptions and navigation path to child pages from a parent page.
+#### When to use 
+- Default option for providing succinct descriptions and navigation paths to child pages from a parent page.
 - Ideally, when an even number of these components can be featured together.
 
-##### When other options are better 
-- When content requires imagery. Suggest using the 50/50 image and text or 25/27 image and text component instead.
+#### When other options are better 
+- When the number of components would be better featured in multiples of 3. Suggest using third-width link blob.
+- When content requires imagery. Suggest using the [50/50 image and text]({{site.baseurl}}//page-components/50-50.html) or [25/75 image and text]({{site.baseurl}}//page-components/25-75.html) components instead.
```

Comment:
It looks like there's an extra slash after `{{site.baseurl}}` in these links (and the ones on the third-width page.)